CONTRIBUTING.md

```diff
@@ -1,69 +1,138 @@
-# Fork Status
+# Contribution guidelines
 
-## Pull Requests
+First off, thank you for considering contributing to ratatui.
 
-**All** pull requests opened on the original repository have been imported. We'll be going through any open PRs in a timely manner, starting with the **smallest bug fixes and README updates**. If you have an open PR make sure to let us know about it on our [discord](https://discord.gg/pMCEU9hNEj) as it helps to know you are still active.
+If your contribution is not straightforward, please first discuss the change you wish to make by
+creating a new issue before making the change.
```

Comment:
A discussion may be more appropriate in some cases.

Reply:
There's a button to convert an issue to a discussion, but not the other way around (you can create a new issue that references a discussion however). There's also a stronger linkage in the PR list to issues. I think maybe I'll rewrite this to be issue / discussion / discord chat.